components/camel-hazelcast/src/main/java/org/apache/camel/component/hazelcast/list/HazelcastListConsumer.java

```diff
@@ -16,25 +16,51 @@
  */
 package org.apache.camel.component.hazelcast.list;
 
-import com.hazelcast.collection.IList;
-import com.hazelcast.core.HazelcastInstance;
+import java.util.UUID;
+
 import org.apache.camel.Consumer;
 import org.apache.camel.Endpoint;
 import org.apache.camel.Processor;
 import org.apache.camel.component.hazelcast.HazelcastDefaultConsumer;
 import org.apache.camel.component.hazelcast.listener.CamelItemListener;
 
+import com.hazelcast.collection.IList;
```

Comment:
I think this will make the camel codestyle fails, but it's not a problem

Reply:
I updated the import order